tasks/parse-pdr/schemas/select-granules.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
```

Comment:
I think this is in the wrong directory.  Looks like it should be in "tests" instead of "schemas".

Reply:
I moved it to tests but do not seem to remove it from schemas

Reply:
ok done!